.storybook/config.js

```diff
@@ -1,33 +1,34 @@
-import { addDecorator, configure } from '@storybook/react';
-import { setOptions } from '@storybook/addon-options';
-import apolloStorybookDecorator from 'apollo-storybook-react';
-import StoryRouter from 'storybook-react-router';
+import { addDecorator, configure } from '@storybook/react'
+import { setOptions } from '@storybook/addon-options'
+import apolloStorybookDecorator from 'apollo-storybook-react'
+import StoryRouter from 'storybook-react-router'
 
-import typeDefs from 'v2/apollo/schema.graphql';
+import typeDefs from 'v2/apollo/schema.graphql'
 
-import mocks from './mocks';
+import mocks from './mocks'
 
-import { wrapWithThemeProvider } from 'v2/styles/theme';
+import { wrapWithThemeProvider } from 'v2/styles/theme'
 
 addDecorator(
   apolloStorybookDecorator({
     typeDefs,
     mocks,
   })
-);
+)
 
-addDecorator(wrapWithThemeProvider);
+addDecorator(wrapWithThemeProvider)
 
-addDecorator(StoryRouter());
+addDecorator(StoryRouter())
 
 setOptions({
   name: 'Are.na',
   url: 'https://www.are.na',
-  sortStoriesByKind: true,
+  showAddonPanel: false,
```

Comment:
This just collapses the annoying empty `Actions` panel at the bottom of the screen.

Reply:
It still toggleable tho? (I do use it for certain components.) But I did notice some super annoying buggy behavior where it remembers position after you close your dev console.

Reply:
Yup, everything is contained inside of the nav (and the hotkey `A`): 

<img width="246" alt="Screen Shot 2019-05-01 at 1 56 56 PM" src="https://user-images.githubusercontent.com/236943/57042265-0d9ebd80-6c19-11e9-813e-613a77090440.png">

(

Reply:
Ooo lovely